### PR TITLE
feat: add enabled option

### LINKED ISSUE #555: Separate dev and storybook server?
Apologies if this has been answered somewhere, but I was unable to find an answer. As a side note, the docs appear to be broken as well so I couldn't find anything there either.

I was wondering if it's possible to run the dev and storybook servers separately? There are times I don't want Storybook to start up, but because this module automatically starts up when running `yarn dev`, for example, I don't have a choice.

### PR DESCRIPTION
**Pull request by huang-julien**
Fixes #555 and fixes https://github.com/storybook-vue/storybook-nuxt/issues/63.

Hi :wave: this PR adds an `enabled` option. 

This can be useful for layers.
Some users have project A, B with A extending B that has storybook but is not needed in A. This can improve dev server startup as storybook can be quite heavy to load on bigger projects.

### CODE REVIEW

**Comment by tobiasdiez** [comment]
Good idea, this would also resolve https://github.com/nuxt-modules/storybook/issues/555, right?

I'm not so sure about the name of the option. Currently, the module indeed only starts the storybook server - but that may change in the future. So I would prefer a name that reflects that only the storybook dev server can be disabled by this option. Tailwind has it under [viewer](https://tailwindcss.nuxtjs.org/getting-started/configuration#viewer). Maybe just also use `viewer` here for storybook? Or do you have better suggestion?

**Comment by huang-julien** [comment]
oh indeed.
About the name, i'm not convinced that `viewer` is a good name. This option just prevent the whole module from being run. `viewer` would fit an option that control if the module adds the storybook tab or not in the devtools.
I don't really have any other suggestion, but i think some other modules are also using `enabled`

**Comment by tobiasdiez** [comment]
Okay, if more fine-grained control is needed in the future, we can add another `viewer/devserver/whatever` option in the future. 

Could you please add a short description of the option to https://github.com/nuxt-modules/storybook/blob/main/docs/content/1.getting-started/2.options.md (perhaps with your use case of layers as motivation). Otherwise this looks good from my side.

File: src/module.ts [hunk 1]
```diff
@@ -33,6 +33,12 @@ export interface ModuleOptions {
    * @default nuxt.options.logLevel
    */
   logLevel: LogLevel
+
+  /**
+   * Eanble the module
```
Comment (tobiasdiez) [inline src/module.ts:38]: typo

**Comment by tobiasdiez** [comment]
Thanks!

**Pull request merged**